README.md

```diff
@@ -96,25 +96,37 @@ Inside the development container perform the following steps.
 
     mkdir -p /app/data
```

Comment:
nit: this command is off -- will fail if not ran as root
should be:
 `sudo mkdir -p /app/data`
 `sudo chown $USER -r /app`

Reply:
These are meant to be run in the container which is root as is set up

Reply:
NVM -- closing -- this is run within a container